### ISSUE TITLE
Build failure on Ubuntu 16.04

### ISSUE THREAD

**Issue by AssRap3r** [mbruel/ngPost#11]
```
sneed@rootserver:~/ngPost/src$ make
/usr/lib/x86_64-linux-gnu/qt5/bin/uic hmi/AboutNgPost.ui -o ui_AboutNgPost.h
/usr/lib/x86_64-linux-gnu/qt5/bin/uic hmi/AutoPostWidget.ui -o ui_AutoPostWidget.h
/usr/lib/x86_64-linux-gnu/qt5/bin/uic hmi/MainWindow.ui -o ui_MainWindow.h
/usr/lib/x86_64-linux-gnu/qt5/bin/uic hmi/PostingWidget.ui -o ui_PostingWidget.h
g++ -c -m64 -pipe -O2 -std=c++1y -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION="3.2" -DQT_NO_DEBUG_OUTPUT -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o NgPost.o NgPost.cpp
NgPost.cpp: In member function ‘bool NgPost::parseCommandLine(int, char**)’:
NgPost.cpp:524:10: warning: unused variable ‘res’ [-Wunused-variable]
     bool res = parser.parse(args);
          ^
g++ -c -m64 -pipe -O2 -std=c++1y -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION="3.2" -DQT_NO_DEBUG_OUTPUT -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o NntpConnection.o NntpConnection.cpp
g++ -c -m64 -pipe -O2 -std=c++1y -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION="3.2" -DQT_NO_DEBUG_OUTPUT -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o PostingJob.o PostingJob.cpp
g++ -c -m64 -pipe -O2 -std=c++1y -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION="3.2" -DQT_NO_DEBUG_OUTPUT -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o AboutNgPost.o hmi/AboutNgPost.cpp
hmi/AboutNgPost.cpp: In constructor ‘AboutNgPost::AboutNgPost(NgPost*, QWidget* ’:
hmi/AboutNgPost.cpp:31:42: error: ‘setWindowFlag’ was not declared in this scope
     setWindowFlag(Qt::FramelessWindowHint);
                                          ^
Makefile:497: recipe for target 'AboutNgPost.o' failed
make: *** [AboutNgPost.o] Error 1



```

Using Ubuntu 16.04.6 LTS

**Comment by mbruel** [comment]
what version of Qt is that 5.7?
check if you can't get at least the 5.9 (LTS) if not the 5.12 (LTS also)
apt-get update
apt-get upgrade

Otherwise I've pushed a fix.
pull and try again

**Comment by AssRap3r** [comment]
latest available in the repos is 5.5.1

AboutNgPost is successful but then fails on something different

```
nntp/NntpFile.cpp: In member function ‘void NntpFile::writeToNZB(QTextStream&, const char*)’:
nntp/NntpFile.cpp:131:33: error: ‘currentSecsSinceEpoch’ is not a member of ‘QDateTime’
                << " date=\"" << QDateTime::currentSecsSinceEpoch() << "\""
                                 ^
Makefile:540: recipe for target 'NntpFile.o' failed
make: *** [NntpFile.o] Error 1

```

I'll probably stick to using the AppImage, seems to run without issue

**Comment by mbruel** [comment]
alright, you should now be able to compile it with Qt v5.5.1 (I've tested on my environment)
the appImage is using Qt v5.12.6, it could be best using it but that's your choice ;)
